src/app/profile/profile.component.ts

```diff
@@ -9,14 +11,13 @@ import { Item } from '../shared/models/item.model';
   styleUrls: ['./profile.component.scss']
 })
 export class ProfileComponent implements OnInit {
-  id: number;
-  student: Student;
-  ready: boolean = false;
+  private id: number;
+  private student: Student;
+  private ready: boolean = false;
```

Comment:
I think by default, boolean values are already `false`, so you don't have to declare it as false.